package-lock.json

```diff
@@ -1,35 +1,42 @@
 {
   "name": "name-examination-upgrade",
-  "lockfileVersion": 3,
+  "lockfileVersion": 2,
```

Comment:
I think the lockfile version depends on the version of Node... so which version of Node is to be used? Could  you add some notes to the README.md file with some details on tools required?

Reply:
idk what happened here...I'm using 16.14.2, as discussed. I've included this in the 'engines' in package.json and i'll add it to the README.md as well.